include/wx/cmndata.h

```diff
@@ -170,20 +171,25 @@ class WXDLLIMPEXP_CORE wxPrintDialogData: public wxObject
     void SetMinPage(int v) { m_printMinPage = v; }
     void SetMaxPage(int v) { m_printMaxPage = v; }
     void SetNoCopies(int v) { m_printNoCopies = v; }
+    void SetCurrentPage(int v) { m_printCurrentPage = v; }
     void SetAllPages(bool flag) { m_printAllPages = flag; }
     void SetSelection(bool flag) { m_printSelection = flag; }
     void SetCollate(bool flag) { m_printCollate = flag; }
     void SetPrintToFile(bool flag) { m_printToFile = flag; }
+    void SetSelectCurrentPage(bool flag) { m_printSelectCurrentPage = flag; }
 
     void EnablePrintToFile(bool flag) { m_printEnablePrintToFile = flag; }
     void EnableSelection(bool flag) { m_printEnableSelection = flag; }
     void EnablePageNumbers(bool flag) { m_printEnablePageNumbers = flag; }
     void EnableHelp(bool flag) { m_printEnableHelp = flag; }
+    void EnableCurrentPage(bool flag) { m_printEnableCurrentPage = flag; }
 
     bool GetEnablePrintToFile() const { return m_printEnablePrintToFile; }
     bool GetEnableSelection() const { return m_printEnableSelection; }
     bool GetEnablePageNumbers() const { return m_printEnablePageNumbers; }
     bool GetEnableHelp() const { return m_printEnableHelp; }
+    bool GetEnableCurrentPage() const { return m_printEnableCurrentPage; }
+    bool GetSelectCurrentPage() const { return m_printSelectCurrentPage; }
```

Comment:
More importantly, I'd move this one to a different function group, together with `GetSelection()` -- as it's a completely kind of flag (this one is user input and not an option set by the program).

Reply:
Ok, changed